packages/contracts/test/core/V3/GenArt721CoreV3_ProjectConfigure.test.ts

```diff
@@ -1242,6 +1243,270 @@ for (const coreContractName of coreContractsToTest) {
+            .getProjectScriptCompressed(""),
+          "Must input non-empty string"
+        );
+      });
+      it("uploads and recalls a single-byte pre-compressed script", async function () {
+        const config = await loadFixture(_beforeEach);
+        const compressedScript = await config.genArt721Core
+          ?.connect(config.accounts.artist)
+          .getProjectScriptCompressed("0");
+        await config.genArt721Core
+          .connect(config.accounts.artist)
+          .addProjectScriptCompressed(config.projectZero, compressedScript);
+        const script = await config.genArt721Core.projectScriptByIndex(
+          config.projectZero,
+          0
+        );
+        expect(script).to.equal("0");
+      });
+
+      it("uploads and recalls an short script < 32 bytes", async function () {
+        const config = await loadFixture(_beforeEach);
+        const targetScript = "console.log(hello world)";
+        const compressedTargetScript = await config.genArt721Core
+          ?.connect(config.accounts.artist)
+          .getProjectScriptCompressed(targetScript);
+        await config.genArt721Core
+          .connect(config.accounts.artist)
+          .addProjectScriptCompressed(
+            config.projectZero,
+            compressedTargetScript
+          );
+        const script = await config.genArt721Core.projectScriptByIndex(
+          config.projectZero,
+          0
+        );
+        expect(script).to.equal(targetScript);
+      });
+
+      it("uploads and recalls chromie squiggle script", async function () {
+        const config = await loadFixture(_beforeEach);
+        const compressedSquiggleScript = await config.genArt721Core
+          ?.connect(config.accounts.artist)
+          .getProjectScriptCompressed(SQUIGGLE_SCRIPT);
+        await config.genArt721Core
+          .connect(config.accounts.artist)
+          .addProjectScriptCompressed(
+            config.projectZero,
+            compressedSquiggleScript
+          );
+        const script = await config.genArt721Core.projectScriptByIndex(
+          config.projectZero,
+          0
+        );
+        expect(script).to.equal(SQUIGGLE_SCRIPT);
+      });
+
+      it("uploads and recalls different script", async function () {
+        const config = await loadFixture(_beforeEach);
+        const compressedSkulptuurScript = await config.genArt721Core
+          ?.connect(config.accounts.artist)
+          .getProjectScriptCompressed(SKULPTUUR_SCRIPT_APPROX);
+
+        await config.genArt721Core
+          .connect(config.accounts.artist)
+          .addProjectScriptCompressed(
+            config.projectZero,
+            compressedSkulptuurScript
+          );
+        const script = await config.genArt721Core.projectScriptByIndex(
+          config.projectZero,
+          0
+        );
+        expect(script).to.equal(SKULPTUUR_SCRIPT_APPROX);
+      });
+
+      it("uploads and recalls 23.95 KB script", async function () {
```

Comment:
you might want to skip this on coverage due to it being a memory-heavy operation. was having some issues with memory limits when testing the library itself.

Reply:
🙏